packages/datadog_flutter_plugin/ios/Classes/DatadogRumPlugin.swift

```diff
@@ -375,6 +375,18 @@ public class DatadogRumPlugin: NSObject, FlutterPlugin {
         return usrMember
     }
 
+    func rumSessionStarted(_ sessionId: String, _ sampled: Bool) {
```

Comment:
maybe it is worth to rename `sampled` here as well to be more descriptive? I would personally think its meaning is that session was sampled in.